website/docs/components/feedback/feedback-form-api.md

```diff
@@ -44,11 +44,22 @@ import FeedbackForm from 'intergalactic/feedback-form';
 
 ## FeedbackForm.Notice
 
-The pre-configured component `Notice`.
+The pre-configured `Notice` component.
 
 ```jsx
 import FeedbackForm from 'intergalactic/feedback-form';
 <FeedbackForm.Notice />;
 ```
 
-<script setup>import { data as types } from '@types.data.ts';</script>
\ No newline at end of file
+<script setup>import { data as types } from '@types.data.ts';</script>
+
+## FeedbackForm.FeedbackRating
+
+The pre-configured `FeedbackForm` component for [FeedbackRating](/patterns/feedback-rating/feedback-rating).
```

Comment:
@msereniti Please check if this needs to be fixed because I'm not sure this description matches the component 🙏